templates/style/base.scss

```diff
@@ -447,6 +447,10 @@ div.cratesfyi-package-container {
     h1 {
         margin: 0;
         padding: 15px 0 0 14px;
+
+        &.no-description {
```

Comment:
You don't need the `&` here.

Reply:
Why not? I need to apply it to `<h1 class="no-description">`.

Reply:
Oooooh. I see what you did there! Indeed, my bad!